tests/tests/test_pathfinding.py

```diff
@@ -88,8 +88,8 @@
             {
                 "track": "TA2",
                 "position": 837.033949007362,
-                "geo": {"type": "Point", "coordinates": [-0.387122554630656, 49.4998]},
-                "sch": {"type": "Point", "coordinates": [-0.387122554630656, 49.4998]},
+                "geo": {"type": "Point", "coordinates": [-0.387122554630656, 49.49979999999999]},
+                "sch": {"type": "Point", "coordinates": [-0.387122554630656, 49.49979999999999]},
```

Comment:
This *should* fall within the range of `pytest.approx`, I'll try to figure out why it doesn't

Reply:
I wrote a new function to fix this, tell me if it's okay for you !